### PR TITLE
Feature#70/home page

### PR DESCRIPTION
**Pull request by juliocabrera820**
# Description
- We've added the home page with the following sections:
  -  Navbar
  -  Hero
  - Product categories
  - Featured products
  - New products
  - Offers
  - Footer

- We've created a new folder structure for our styles, due to we were able to define an SASS architecture

### CODE REVIEW

**Review by brayandiazc — APPROVED** [review]
Approve

**Comment by brayandiazc** [comment]
Approve

**Review by xFerchoVJ — APPROVED** [review]
LGTM!

**Review by oramasangel — APPROVED** [review]
lgtm

File: local_sales/app/views/customers/_categories.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h1 class="text-4xl font-extrabold text-gray-900 sm:text-3xl">Categories</h1>
+<div class="mt-6 grid grid-cols-2 gap-y-10 sm:grid-cols-2 sm:gap-x-16 lg:grid-cols-2 xl:gap-x-16">
+   <% @categories.each do |category| %>
+     <%= render 'category_card', category: category %>
+   <% end %>
+  </div>
+</div>
```
Comment (xFerchoVJ) [inline local_sales/app/views/customers/_categories.html.erb:7]: Is the openning of this div missing?

File: local_sales/app/views/customers/_category_card.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="group relative">
+  <div class="w-full rounded-md group-hover:opacity-75 lg:h-100 lg:aspect-none">
+    <%= image_tag('category', class: 'w-full h-full object-center object-cover lg:w-full lg:h-full') %>
```
Comment (xFerchoVJ) [inline local_sales/app/views/customers/_category_card.html.erb:3]: As a suggestion add the alt attribute to the image_tag
example:
`<% = image_tag ("image_path", class: "image styles", alt: "Image description")%>`

File: local_sales/app/views/customers/_featured_products.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1 class="text-4xl font-extrabold text-gray-900 sm:text-3xl">Featured products</h1>
+<div class="mt-6 grid sm:grid-cols-2 sm:gap-x-8 xl:grid-cols-3 xl:gap-x-12">
+   <% @featured_products.each do |featured_product| %>
+     <%= render 'product_card', product: featured_product %>
+   <% end %>
+  </div>
+</div>
```
Comment (YilKanda) [inline local_sales/app/views/customers/_featured_products.html.erb:7]: This div, where does it open?

File: local_sales/app/views/customers/_categories.html.erb (hunk 1 above)
Comment (YilKanda) [inline local_sales/app/views/customers/_categories.html.erb:7]: This div, where does it open?

**Review by YilKanda — COMMENTED** [review]
It's important that you describe changes to the PR, so reviewers can quickly identify changes to your code.

File: local_sales/app/views/customers/_featured_products.html.erb (hunk 3 above)
Comment (juliocabrera820) [reply local_sales/app/views/customers/_featured_products.html.erb:7]: This PR is WIP, so we can check all details before assign the *mentor review required* label

File: local_sales/app/views/customers/_hero.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<section class="min-h-screen bg-blue-100 lg:p-32 p-14 text-right" style="background-image: url('https://images.unsplash.com/photo-1606836483111-e08386681a8b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80'); background-size: cover">
+	<h1 class="lg:text-5xl text-xl font-bold text-white mb-14 px-2">Conoce todos los productos que tenemos para ti</h1>
+	<div class="animate-bounce">
+		<a class="lg:py-4 py-2 lg:px-8 px-4 bg-gray-400 hover:bg-gray-300 lg:font-bold text-gray-800 hover:text-gray-900 rounded-lg
+		  transition duration-300">Descubre nuevos productos</a>
```
Comment (YilKanda) [inline local_sales/app/views/customers/_hero.html.erb:5]: Check your indentation

**Review by YilKanda — COMMENTED** [review]
You just have a problem with indentation in a file.

**Review by xFerchoVJ — APPROVED** [review]
Good use of variables in scss, LGTM

**Review by JoseRicardoJimenez — DISMISSED** [review]
NICE!

**Review by oramasangel — APPROVED** [review]
lgtm

**Review by JoseRicardoJimenez — APPROVED** [review]
OK!

**Pull request merged**